main.go

```diff
@@ -152,7 +152,8 @@ func main() {
 	}
 	// +kubebuilder:scaffold:builder
 
-	enforcer := enforcer.NewEnforcer(ctrl.Log.WithName("enforcer"), attesters, grafeasClient, mgr.GetClient())
+	signerList := attester.NewSignerList()
+	enforcer := enforcer.NewEnforcer(ctrl.Log.WithName("enforcer"), attesters, grafeasClient, signerList, mgr.GetClient())
```

Comment:
I think we should be able to remove attesters from here since signerList is replacing the functionality.